src/NzbDrone.Core/Download/DownloadEventHub.cs

```diff
@@ -77,5 +79,23 @@ private void RemoveFromDownloadClient(TrackedDownload trackedDownload)
                 _logger.Error(e, "Couldn't remove item from client {0}", trackedDownload.DownloadItem.Title);
             }
         }
+
+        private void MarkItemAsImported(TrackedDownload trackedDownload)
+        {
+            var downloadClient = _downloadClientProvider.Get(trackedDownload.DownloadClient);
+            try
+            {
+                _logger.Debug("[{0}] Marking download as imported from {1}", trackedDownload.DownloadItem.Title, trackedDownload.DownloadItem.DownloadClient);
+                downloadClient.MarkItemAsImported(trackedDownload.DownloadItem);
+            }
+            catch (NotSupportedException)
+            {
+                _logger.Warn("Marking item as imported is not supported by your download client ({0}).", downloadClient.Definition.Name);
```

Comment:
Hm, I missing this in an earlier review.
We could just change this to _logger.Debug and throw new NotSupportedException in DownloadClientBase, instead of adding a _logger.Debug there.